File: Menu.java
package pck;

import java.util.Scanner;
//----------------------------Menu----------------------------------
public class Menu {
	public static void main(String[] args) {
		Scanner escaner= new Scanner(System.in);
		int opcion=0;
		String nombre="";
		int edad=0;
		while (opcion!=3) {
			opcion=menu();

			if (opcion==1) {
				for (int k=0;k<5;k++) {
					System.out.println("Ingrese su nombre: "+(k+1));
					nombre=escaner.nextLine();
					System.out.println("Ingrese su edad: ");
					edad=Integer.parseInt(escaner.nextLine());;
				}
			}
			else {
				if (opcion==2) {
					System.out.println("Su nombre es: " +nombre);
					System.out.println("Su edad es: " +edad);
				}
				else {
					if (opcion==3) {
						System.out.println("Saliendo del programa...");

					} 
					else {
						System.out.println("Numero no valido");
					}
				}
			}
		}
	}
	public static int menu() {
		Scanner escaner= new Scanner(System.in);
		System.out.println("===== MENÚ =====");
		System.out.println("Opcion 1: Ingresar datos");
		System.out.println("Opcion 2: Mostrar datos");
		System.out.println("Opcion 3: Salir");
		System.out.println("Seleccione una opcion: ");
		int opcion=Integer.parseInt(escaner.nextLine());
		return opcion;
	}
}

File: Menu.py
def mostrar_menu():
    print("\n===== MENÚ PRINCIPAL =====")
    print("1. Mostrar números")
    print("2. Sumar números")
    print("3. Validar contraseña")
    print("4. Salir")
    return int(input("Seleccione una opción: "))


def mostrar_numeros(n):
    for k in range(1, n + 1):
        print(k)


def sumar_numeros():
    suma = 0
    num = -1

    while num != 0:
        num = int(input("Ingrese un número (0 para terminar): "))
        suma += num

    return suma


def validar_password():
    password_correcta = "Contraseña"

    while True:
        password = input("Ingrese la contraseña: ")
        if password == password_correcta:
            break

    print("Acceso concedido")


opcion = 0

while opcion != 4:
    opcion = mostrar_menu()

    if opcion == 1:
        n = int(input("Ingrese un número: "))
        mostrar_numeros(n)

    elif opcion == 2:
        print("La suma total es:", sumar_numeros())

    elif opcion == 3:
        validar_password()

    elif opcion == 4:
        print("Saliendo del programa...")

    else:
        print("Opción inválida")
